doc/api/modules.md

```diff
@@ -552,16 +552,20 @@ added: v0.1.13
 
 * {Function}
 
-Used to import modules, `JSON` and local files. Modules can be imported from `node_modules`. Local modules and JSON files can be imported using the pattern `'./'`.
+Used to import modules, `JSON`, and local files. Modules can be imported
+from `node_modules`. Local modules and JSON files can be imported using
+a relative path (e.g. `./`, `./foo`, `./bar/baz`, `../foo`) that will be
```

Comment:
Should we perhaps also note that path-resolution is OS independent (given that we are providing examples in Linux/Unix flavor)?

Reply:
@lundibundi I guess that would be a good addition but it could also land in a separate PR?